src/supporting-data/keywords/components/entry/styles/keywords-graph.module.scss

```diff
@@ -0,0 +1,32 @@
+.graph {
+  position: relative;
+  width: 500px;
+  height: calc(50vh - 100px);
+}
+
+.container {
+  width: 500px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: calc(50vh - 100px);
+  position: absolute;
+
+  :global(.node) {
+    width: min-content;
+  }
+}
+
+.keyword-node {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  text-align: center;
+  flex-wrap: wrap;
+}
+
+.arrows {
```

Comment:
Is there a way to space the arrows out a little when two point at one node? Thinking of the 2Fe-2S node at the bottom of this diagram:
<img width="558" alt="Screenshot 2022-06-16 at 08 26 30" src="https://user-images.githubusercontent.com/4357962/174017474-a7fe50f9-4d7d-4725-b9af-42d2cdf7a8cf.png">